src/components/review-blurb.jsx

```diff
@@ -0,0 +1,32 @@
+import Database from '../database';
+import Loadable from './loadable';
+
+function ReviewBlurb({card: {url}}) {
+  const reviewNumber = url.split('/').pop();
+
+  return (
+    <span className='review-blurb'>
+      <a className='blurb-number-link'
+        target='_blank'
+        href={url}
+        >
+        <span className='blurb-number'>{reviewNumber}</span>
+      </a>
+    </span>
+  );
+}
+
+function ReviewBlurbShell(props) {
+  const {card, primaryRepoName, context} = props;
```

Comment:
Do it like above component?
Also pull repoOwner and repoName only if it isn't too ugly.

Reply:
`pull repoOwner and repoName` that doesn't work since `card` is needed within the function.